docs/UserGuide.md

```diff
@@ -9,6 +9,28 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
   {:toc}
 
 --------------------------------------------------------------------------------------------------------------------
+1. [Quick start](#quick-start)
+2. [Features](#features)
+   * [help](#viewing-help-help)
+   * [questionnaire](#viewing-risk-assessment-questionnaire-questionnaire)
+   * [riskprofile](#adds-risk-profile-level-to-a-contactriskprofile)
+   * [add](#adding-a-person-add)
+   * [clone](#cloning-a-person--clone)
+   * [delete](#deleting-a-person--delete)
+   * [list](#list-out-all-contacts--list)
+   * [edit](#editing-a-person--edit)
+   * [find](#locating-persons-by-name-find)
+   * [undo](#undoing-a-command--undo)
+   * [redo](#redoing-an-undo-command--redo)
+   * [sort](#sorting-contacts--sort)
+   * [calendar](#opening-calendar-window--calendar)
+   * [clear](#clearing-all-entries--clear)
+   * [exit](#exiting-the-program--exit)
+3. [FAQ](#faq)
+4. [Common Questions](#common-questions)
+5. [Known issues](#known-issues)
```

Comment:
Since the (Q) in the Common Questions is capitalized, how about making the (i) in the Known issues capitalized too?

Reply:
I agree with Daphne on the above, I'll merge this first so we can all edit the UG